src/testers.jl

```diff
@@ -228,6 +228,7 @@ function frule_test(f, xẋs::Tuple{Any, Any}...; rtol::Real=1e-9, atol::Real=1e
     end
     res = frule((NO_FIELDS, deepcopy(ẋs)...), f, deepcopy(xs)...; deepcopy(fkwargs)...)
     res === nothing && throw(MethodError(frule, typeof((f, xs...))))
+    res isa Tuple || error("The frule should return (y, ∂y), not $res.")
```

Comment:
this adds a hopefully useful bit of info

Reply:
LGTM